test/hash.test.ts

```diff
@@ -24,11 +24,13 @@ describe('hash', () => {
     try {
       await hash(filePath);
```

Comment:
Not related to this PR, but shouldn't we have a test failure between lines 25 and 26, since we only test for the correct exception, if and only if we throw an error. So this might be a false negative if we introduce new logic in the feature.